lib/seahorn/Bmc.cc

```diff
@@ -216,6 +216,14 @@ namespace seahorn
     return m_model.eval (v, complete);
   }
   
+  DenseSet<const BasicBlock*> BmcTrace::getRegion()
```

Comment:
I was aware of the performance issue. I thought it was not a big deal since it is only called once and not in critical part. But since you brought this up, I'll change it.